cmd/buildah/containers.go

```diff
@@ -239,6 +240,13 @@ func outputContainers(store storage.Store, opts containerOptions, params *contai
 }
 
 func containerOutputUsingTemplate(format string, params containerOutputParams) error {
+	matched, err := regexp.MatchString("{{.*}}", format)
```

Comment:
This still does not match the format suggested before.  Perhaps something like
```
matched, err := regexp.MatchString(".*{{.*}}", format)
```
But add some tests of valid formats and invalid formats.

Reply:
Don't need change. It works.

```
➜  buildah git:(containers-format) sudo ./buildah containers --format "containerID: {{.ContainerID}}"
containerID: e477836657bba9fd76b80fd71c13318dc18a78711102a32b110e72d37187602e
containerID: 09d1845b52485ce3bc431ff8e57b1e4910723d96af6223bf3a421ed6b48a5d6d
containerID: b025ebe4b1d94887e876f2ebc7e410396f5fe4780ec1f80437d36c39cc3189d8
containerID: 5666da5d29003faecbaf44c951ee465402aa2c500e4659b60767ebb7c94b087e
```

Reply:
Excellent.

Reply:
Is this new code OK with this:

sudo ./buildah containers --format "containerID: {{}}"

i.e. does it hit the error that you have at line 247 or does it do something else?